share/spack/gitlab/cloud_pipelines/stacks/e4s/spack.yaml

```diff
@@ -325,6 +328,7 @@ spack:
   - amrex +rocm amdgpu_target=gfx90a
   - caliper +rocm amdgpu_target=gfx90a
   - chai +rocm amdgpu_target=gfx90a
+  - chapel +rocm amdgpu_target=gfx90a
```

Comment:
@eugeneswalker you missed one.

Note that `chapel@2.1 +rocm` only supports ROCm versions <= 5.4, which are all `deprecated`

Reply:
Yes, my bad! Missed that. Just pushed a commit to comment this out as well!